pkgs/applications/terminal-emulators/mlterm/default.nix

```
@@ -18,9 +19,25 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig autoconf wrapGAppsHook ];
   buildInputs = [
-    libX11 gdk-pixbuf.dev cairo libXft gtk3 vte
-    harfbuzz fribidi m17n_lib openssl libssh2
-    fcitx ibus uim
+    libX11
+    gdk-pixbuf.dev
+    cairo
+    libXft
+    gtk3
+    harfbuzz
+    fribidi
+  ] ++ lib.optional (!stdenv.isDarwin) [
```

this should be `optionals` since the second argument is a list
```suggestion
  ] ++ lib.optionals (!stdenv.isDarwin) [
```

---

Odd that it worked anyways. The Linux drv had its arguments in a different order but was the same otherwise.